src/bp/core/config/config-loader.ts

```diff
@@ -34,7 +34,7 @@ export class ConfigProvider {
     @inject(TYPES.ObjectCache) private cache: ObjectCache
   ) {
     this.cache.events.on('invalidation', async key => {
-      if (key === 'object::data/global/botpress.config.json') {
+      if (key === 'object::data/global/botpress.config.json' || key === 'file::data/global/botpress.config.json') {
```

Comment:
Difference between `file:: `, `object::` and `buffer::`? One is a file on the filesystem and the others an object in the cache (or maybe the database)?

Reply:
file:: comes from the chokidar watcher, object:: and buffer:: are from bpfs.readFileAsBuffer and bpfs.readFileAsObject